src/main/java/org/infernalstudios/shieldexp/ShieldExpansion.java

```diff
@@ -14,37 +14,47 @@
  */
 package org.infernalstudios.shieldexp;
 
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-
+import org.infernalstudios.shieldexp.events.ClientEvents;
+import org.infernalstudios.shieldexp.events.ShieldExpansionEvents;
+import org.infernalstudios.shieldexp.init.Config;
+import org.infernalstudios.shieldexp.init.ItemsInit;
+import org.infernalstudios.shieldexp.init.NetworkInit;
+import org.infernalstudios.shieldexp.init.ShieldDataLoader;
+import net.minecraftforge.api.distmarker.Dist;
 import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.eventbus.api.IEventBus;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.DistExecutor;
+import net.minecraftforge.fml.ModLoadingContext;
 import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.config.ModConfig;
 import net.minecraftforge.fml.event.lifecycle.FMLClientSetupEvent;
 import net.minecraftforge.fml.event.lifecycle.FMLCommonSetupEvent;
 import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;
-import org.infernalstudios.shieldexp.events.ClientEvents;
-import org.infernalstudios.shieldexp.events.ShieldExpansionEvents;
 
-@Mod("shieldexp")
+@Mod(ShieldExpansion.ID)
 public class ShieldExpansion {
-	public static final String NAME = "Shield Expansion";
-	public static final String MOD_ID = "shieldexp";
-	public static final Logger LOGGER = LogManager.getLogger(NAME);
-
-	public ShieldExpansion() {
-		FMLJavaModLoadingContext.get().getModEventBus().addListener(this::commonSetup);
-		FMLJavaModLoadingContext.get().getModEventBus().addListener(this::clientSetup);
+    public static final String ID = "shieldexp";
```

Comment:
Please keep this constant as `MOD_ID` so it's consistent across all of our projects.